Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt

```diff
@@ -2,48 +2,138 @@
+39.	has-ansi (https://github.com/chalk/has-ansi)
+40.	hash-base (https://github.com/crypto-browserify/hash-base)
+41.	hawk (https://github.com/hueniverse/hawk)
+		Includes:CryptoJS
+42.	hoek (https://github.com/hapijs/hoek)
+		Includes:Deep-eql
+43.	http-signature (https://github.com/joyent/node-http-signature)
+44.	inherits (https://github.com/isaacs/inherits)
+45. isarray (https://github.com/juliangruber/isarray/)
+46.	is-buffer (https://github.com/feross/is-buffer)
+47.	is-my-json-valid (https://github.com/mafintosh/is-my-json-valid)
+48.	is-property (https://github.com/mikolalysenko/is-property)
+49.	isstream (https://github.com/rvagg/isstream)
+50.	is-typedarray (https://github.com/hughsk/is-typedarray)
+51. Jenkins logo
+52. jodid25519 (https://github.com/meganz/jodid25519)
+		Includes:almond
+53.	jsbn (https://github.com/andyperlitch/jsbn)
+54.	json-edm-parser (https://github.com/yaxia/json-edm-parser)
+55.	jsonparse (https://github.com/creationix/jsonparse)
+56.	jsonpointer (https://github.com/janl/node-jsonpointer)
+57.	json-schema (https://github.com/kriszyp/json-schema)
+58.	json-stringify-safe (https://github.com/isaacs/json-stringify-safe)
+59.	jsprim (https://github.com/davepacheco/node-jsprim)
+60.	kind-of (https://github.com/jonschlinkert/kind-of)
+61.	lazy-cache (https://github.com/jonschlinkert/lazy-cache)
+62.	lodash (https://github.com/lodash/lodash)
+		Includes:Sizzle CSS Selector Engine
+		Includes:Webpack
+63.	longest (https://github.com/jonschlinkert/longest)
+64.	lru-cache (https://github.com/isaacs/node-lru-cache)
+65.	md5.js (https://github.com/crypto-browserify/md5.js)
+66.	mime-db (https://github.com/jshttp/mime-db)
+67.	mime-types (https://github.com/broofa/node-mime)
+68.	minimatch (https://github.com/isaacs/minimatch)
+69.	minimist (https://github.com/substack/minimist)
+70.	mockery (https://github.com/mfncooper/mockery)
+71.	optimist (https://github.com/substack/node-optimist)
+72. oauth-sign (https://github.com/request/oauth-sign)
+73.	pinkie (https://github.com/floatdrop/pinkie)
+74.	pinkie-promise (https://github.com/floatdrop/pinkie-promise)
+75.	process-nextick-args (https://github.com/calvinmetcalf/process-nextick-args)
+76.	pseudomap (https://github.com/isaacs/pseudomap)
+77.	punycode (https://github.com/bestiejs/punycode.js)
+78.	q (https://github.com/kriskowal/q)
+79.	qs (https://github.com/ljharb/qs)
+		Includes:io.js
+80.	readable-stream (https://github.com/nodejs/readable-stream)
+81.	request (https://github.com/request/request)
+82.	repeat-string (https://github.com/jonschlinkert/repeat-string)
+83.	right-align (https://github.com/jonschlinkert/right-align)
+84. safe-buffer (https://github.com/feross/safe-buffer)
+85.	sax (https://github.com/isaacs/sax-js)
+86.	semver (https://github.com/npm/node-semver)
+87.	shelljs (https://github.com/arturadib/shelljs)
+		Includes:wrench-js
+88. sntp (https://github.com/hueniverse/sntp)
+89.	source-map (https://github.com/mozilla/source-map)
```

Comment:
source-map (https://github.com/mozilla/source-map) is mentioned twice.

Reply:
# resolved